cms/envs/common.py

```diff
@@ -834,6 +834,10 @@
     'notes',
     'schoolyourself_review',
     'schoolyourself_lesson',
+
+    # Google Drive embedded components
```

Comment:
Just because I feel lIke I need to say *something* :)  Would it be possible to include a bit more context here, perhaps with URL references as above?